srv/GraspPlanning.srv

```diff
@@ -0,0 +1,27 @@
+# Requests that grasp planning be performed for the target
+# returns a list of grasps to be tested and executed
+
+# the planning group used
+string group_name
+
+# the object to be grasped
+CollisionObject target
+
+# the name that the support surface (e.g. table) has in the collision map
+# can be left empty if no name is available
+string collision_support_surface_name
+
+# an optional list of grasps to be evaluated by the planner
+Grasp[] grasps_to_evaluate
+
+# an optional list of obstacles that we have semantic information about
+# and that can be moved in the course of grasping
+CollisionObject[] movable_obstacles
```

Comment:
Right. I think the  potential usage you described is already encompassed in the __allowed_touch_objects__ entry in the [Grasp.msg](http://docs.ros.org/api/moveit_msgs/html/msg/Grasp.html) messages in this service definition; in fact they even have the same description.  I think we should avoid having entries with similar purpose as that could make the interface confusing.